scripts/hid_configurator/README.rst

```diff
@@ -82,7 +82,15 @@ Complete the following steps:
        In versions earlier than 5.44, the HID device attached by BlueZ could obtain wrong VID and PID values (ignoring values in Device Information Service), which would stop HIDAPI from opening the device.
        In versions earlier than 5.56, the HID device attached by BlueZ might provide incomplete HID feature report on get operation.
 
-   Additionally, to call the Python script on Linux without root rights, install the provided udev rule :file:`99-hid.rules` file by copying it to :file:`/etc/udev/rules.d` and replugging the device.
+#. Additionally, to call the Python script on Linux without root rights, install the provided udev rule :file:`99-hid.rules` file by copying it to :file:`/etc/udev/rules.d` and replugging the device.
+  If you want to connect to your device which has different Vendor or Product ID you can run script with root permissions or add new entry to :file:`99-hid.rules` file with your Vendor and Product ID and install it by coping it to :file:`/etc/udev/rules.d` and replugging the device.
```

Comment:
`... has different Vendor or Product ID than the ones specified in the file ...`